src/Chainweb/BlockHeader/Genesis.hs

```diff
@@ -90,6 +90,8 @@ genesisTime FastTimedCPM{} = BlockCreationTime epoch
 genesisTime Development = BlockCreationTime . Time $ TimeSpan 1563388117613832
```

Comment:
We need these times to be human-readable in code

Reply:
Hence the comment above it. In particular, this is a field that goes into a `BlockHeader`, which is a number-based `BlockCreationTime`. These are often used for comparisons.

Reply:
Obviously, but you can create these here using a utility function.

Reply:
We need a human-verifiable value for these constants as they constitute serious business risk if they are wrong. Either
1) Offer a solution to generate them performantly or
2) Write a unit test that proves they are correct.

Reply:
Also notice that the comment for `Testnet02` has gone stale because it is the same value as `Development` but the comment says something different.  Furthermore, none of these comments are in UTC and they don't specify the timezone that they are in.  So yeah...see the new PR #581 I just opened to be merged into this one.